src/lib/api.tsx

```diff
@@ -42,19 +51,19 @@ export const getDiscussion = (
   };
   const params = objAsParams(apiOpts);
 
-  const url = joinUrl([baseURL, "discussion", shortUrl, params]);
+  const url = joinUrl([options.baseUrl, "discussion", shortUrl]) + params;
```

Comment:
By passing `params` through `joinUrl` it was adding an extra '/' at the end of the path and the discussion api doesn't handle trailing slashes and returns a 404